### ISSUE TITLE
rpmbuild: --build-in-place: unknown option

### ISSUE THREAD

**Issue by titansmc** [acassen/keepalived#897]
Hi,
I am trying to build the packages for CentOS, but things are not to clear for me, are there instructions for this? 
I am doing:

- Installing all packages (build esentials, autoconf.....
- Clone repo
- ./build_setup
- Copy source to ~/rpmbuild/SOURCES/keepalived-2.0.1.tar.gz
- ./configure
- make rpm

And to get make not failing, I have to remove this build-in-place

```
[root@haproxy03 keepalived]# make rpm
rpmbuild -ba --build-in-place keepalived.spec
rpmbuild: --build-in-place: unknown option
make: *** [rpm] Error 1
```

Is there anything I am missing? And, if this is the correct procedure, can we add it somewhere in the documentation?

Best.

**Comment by pqarmitage** [comment]
It appears that the version of rpmbuild on CentOS doesn't yet support `--build-in-place`. I have added commit 413171e which checks whether rpmbuild supports `--build-in-place` and if not, it creates a tarball, copies it to the rpmbuild/SOURCES directory, and executes rpmbuild using that.

The process to build the rpm files is:
1. Clone the git repo
2. ./build_setup
3. ./configure
4. make rpm

I have also updated the INSTALL file to include these instructions.
